package.json

```diff
@@ -14,6 +14,7 @@
         "@angular/core": "11.2.10",
         "@angular/forms": "11.2.10",
         "@angular/localize": "11.2.10",
+        "@angular/material": "^11.2.9",
```

Comment:
can we please avoid the additional dependency?

Reply:
we use bootstrap and not material in angular. Having both libraries is overhead and will lead to inconsistencies

Reply:
I removed this.